flow/ProtocolVersions.cmake

```diff
@@ -93,3 +93,4 @@ set(FDB_PV_CLUSTER_ID_SPECIAL_KEY               "0x0FDB00B072000000LL")
 set(FDB_PV_BLOB_GRANULE_FILE_LOGICAL_SIZE       "0x0FDB00B072000000LL")
 set(FDB_PV_BLOB_RANGE_CHANGE_LOG                "0x0FDB00B072000000LL")
 set(FDB_PV_GC_TXN_GENERATIONS                   "0x0FDB00B073000000LL")
+set(FDB_PV_MUTATION_CHECKSUM                    "0x0FDB00B073000000LL")
```

Comment:
Not sure if we can do this, as the old 7.3 client doesn't have `MutationRef:: checksum`, thus is incompatible with the server. It's probably safe to use 7.4 as the first version with this feature. If we want this in 7.3, the deployment needs to roll out new 7.3 client bindings first, before the server side upgrade.